website/src/releases/2021-04/release-v19.mdx

```diff
@@ -46,14 +46,18 @@ The equivalent mapping from `constraint` to `max` is as following:
 
 ## Text
 
+{/* <!-- vale off --> */}
+
 For the `<Text.Headline>` and `<Text.Subheadline>` components, the deprecated `elementType` prop has been removed. Now only the `as` prop is used instead.
 
+{/* <!-- vale on --> */}
+
 ```diff
 - <Text.Headline elementType="h2" intlMessage={message} />
 + <Text.Headline as="h2" intlMessage={message} />
 ```
 
-For the `<Text.Body>` and `<Text.Detail>` components, the deprecated `isInline` prop has been removed. Now only the `as` prop can be used instead.
+For the `<Body>` and `<Detail>` components, the deprecated `isInline` prop has been removed. Now only the `as` prop can be used instead.
```

Comment:
Perhaps we can use the `vale` directive here as well:
```suggestion
{/* <!-- vale off --> */}

For the `<Text.Body>` and `<Text.Detail>` components, the deprecated `isInline` prop has been removed. Now only the `as` prop can be used instead.

{/* <!-- vale on --> */}
```

Reply:
Ah missed that, thanks

Reply:
https://github.com/commercetools/merchant-center-application-kit/pull/2961/commits/4284137818c99c9961549a82ecac0595fb4a8031